packages/connector-logto-email/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import type {
+  CreateConnector,
+  EmailConnector,
+  GetConnectorConfig,
+  SendMessageFunction,
+} from '@logto/connector-kit';
+import { ConnectorType, validateConfig } from '@logto/connector-kit';
+import { HTTPError, got } from 'got';
+
+import { defaultMetadata, defaultTimeout, emailEndpoint } from './constant.js';
+import { grantAccessToken } from './grant-access-token.js';
+import type { LogtoEmailConfig } from './types.js';
+import { logtoEmailConfigGuard } from './types.js';
+
+const sendMessage =
+  (getConfig: GetConnectorConfig): SendMessageFunction =>
+  async (data, inputConfig) => {
+    const config = inputConfig ?? (await getConfig(defaultMetadata.id));
+    validateConfig<LogtoEmailConfig>(config, logtoEmailConfigGuard);
```

Comment:
will the config be seeded in the db directly without user's configuration?

Reply:
YES